src/System.Reflection.Extensions/src/System.Reflection.Extensions.builds

```diff
@@ -3,6 +3,12 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Reflection.Extensions.csproj" />
+    <Project Include="System.Reflection.Extensions.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
```

Comment:
We shouldn't any longer need to build for netstandard1.3 as it should just get picked up form the previous package, for the same reason as the others they should just get picked up automatically from the last stable package.